_assets/js/feature-flag.js

```diff
@@ -0,0 +1,38 @@
+export default function getCookies() {
+    const isNetlifyUser = sessionStorage.getItem('isNetlifyUser') ?? false;
+    const featureFlags = setFeatureFlagCookies(isNetlifyUser);
+    const variant = setVariant();
+
+    return {'variant': variant, 'feature_flags': featureFlags};
+  }
+
+function setVariant() {
+  let variant;
+  // Check if the user already has a variant assigned
+  const userVariant = document.cookie
+      .split("; ")
+      .find((row) => row.startsWith("ab_variant="))
+      ?.split("=")[1];
+
+  if (userVariant) {
+    variant = userVariant;
+  } else {
+    // Assign a random variant if the user doesn't have one
+    variant = Math.random() < 0.5 ? 'A' : 'B';
+    // Set the variant in a cookie for future visits
+    document.cookie = 'ab_variant='+ variant;
+  }
+
+  return variant;
+}
+
+const FEATURE_FLAGS = [
```

Comment:
WDYT about using this same list to assign different a/b test variants, as well?

I guess I'm thinking of "feature" as a singular object in the sense that you could:

1. define a "feature" (e.g., 'laws-and-regs')
2. turn that feature on or off for everyone (100% or 0%)
3. turn that feature on or off for a subset (i.e. 20%, 44%, etc; "a/b testing")
4. opt in/out of the feature as an admin ("feature flag")